addon/components/popover/trigger/index.ts

```diff
@@ -0,0 +1,38 @@
+import { action } from '@ember/object';
+import { next } from '@ember/runloop';
+import Component from '@glimmer/component';
+
+import { normalizeKey } from 'backpack-ember/utils/normalize-keycode';
```

Comment:
Getting an error in the browser when running the dummy app:
```
Could not find module `backpack-ember/utils/normalize-keycode` imported from `@showbie/backpack-ember/components/popover/trigger/index`
```

Reply:
Is the import missing the `@showbie` scope here?

Reply:
Yeah, it was throwing me off because I was getting a TS error in my editor when I added that, but fixing the alias in tsconfig.json fixed it 👍